### PR TITLE
MGDCTRS-1623 fix: "No connectors instances" if there's less connectors than the current page range

### PR DESCRIPTION
**Pull request by kelvah**
JIRA: https://issues.redhat.com/browse/MGDCTRS-1623

I've implemented Stan's proposal of showing the "No results" message when a page > 1 returns no items.
Users can then click on "clear all filters" to go back to the first page unfiltered.

### CODE REVIEW

File: src/app/pages/ConnectorsPage/components/ConnectorInstances/ConnectorInstances.tsx [hunk 1]
```diff
@@ -111,7 +111,9 @@ export const ConnectorInstances: FunctionComponent<ConnectorInstancesProps> = ({
   );
 
   const onRemoveGroup = onClearAllFilters;
-  const isFiltered = namesChips.length > 0;
+  // isFiltered determines if the TableView component will show the "no results" block
+  // instead of the "empty state" when no items are present. See MGDCTRS-1623 for more details.
+  const isFiltered = namesChips.length > 0 || page > 1;
```
Comment (kelvah) [inline src/app/pages/ConnectorsPage/components/ConnectorInstances/ConnectorInstances.tsx:116]: `TableView` shared component [is handling](https://github.com/redhat-developer/app-services-ui-components/blob/028fb603423d4178c038debe237b411ce2e29fb4/src/shared/Table/TableView.tsx#L104) what to show when no items are present.
Setting the `isFiltered` flag also for page > 1 will fix the issue.

File: cypress/e2e/connectors-instances.cy.ts [hunk 2]
```diff
@@ -179,18 +179,62 @@ describe('Connectors page', () => {
     cy.findAllByText('Something went wrong').should('have.length', 0);
   });
 
-  xit('can search for a connector', () => {
+  it('can search for a connector', () => {
```
Comment (kelvah) [inline cypress/e2e/connectors-instances.cy.ts:182]: Took the chance to fix and re-introduce this skipped test.

Comment (gashcrumb) [reply cypress/e2e/connectors-instances.cy.ts:182]: awesome!

**Review by gashcrumb — APPROVED** [review]
This looks great and thanks for re-adding that test!  :+1:

**Pull request merged**